docs/plugin-development/restrictions.md

```diff
@@ -82,6 +82,71 @@ start debugging performance issues is through the Plugin Statistics window,
 which can be found through Plugins > Open Plugin Stats in the dev menu
 (`/xldev`).
 
+## Can my plugin talk to a backend server I run?
+
+Plugins are permitted to communicate with special backend servers, though there
+are certain considerations and requirements that must be met:
+
+- Plugins should take care to send the minimum amount of data necessary to do
+  their job. Whenever feasible, plugins should hash information about the local
+  player (such as the player's Content ID or name) on the client side so that a
+  server-side data breach does not reveal information.
+- Plugins may collect additional non-essential information for telemetry,
+  analytics, or statistical purposes, provided the user is given a chance to
+  review what data is being collected and for what purpose.
+  - Users must opt in to additional telemetry collection, but this may be done
+    as part of a "welcome to this plugin" experience forcing an explicit choice,
+    or via a global Dalamud setting.
+  - Additional data collection must be done for the public interest. That is,
+    the extra information should go back to improvement of the plugin, provide
+    the public with statistics or dashboards, facilitate the creation of curios,
+    or otherwise just improve the game and player experience.
+  - Plugins must use either a pseudo-random identifier (or no identifier) for
```

Comment:
Not commenting on the merits, but on sentence structure:
If the `either` is not in parentheses, then the `(or no identifier)` should also not be. As it stands, the sentence makes no sense if the parenthesised extraneous information is omitted.